cms/check_report_used_space

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2013
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+# - Fernando Garzon, <oscar.fernando.garzon.miguez.ch> 2020
+
+'''
+Probe to check used space.
+'''
+
+import datetime
+import unicodedata
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage, list_rse_attributes
+from rucio.db.sqla import models
+from rucio.db.sqla.session import BASE, get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if BASE.metadata.schema:
+    schema = BASE.metadata.schema + '.'
+
+if __name__ == "__main__":
+    try:
+        status = OK
```

Comment:
`status` isn’t actually used. Consider either explicitly calling `sys.exit()` at the end or removing this.